### PR TITLE
feature(new for you): create /new-for-you

### PR DESCRIPTION
**Pull request by laurabeth**
The type of this PR is: **Feature**

This PR solves [GRO-976]

### Description

this pr creates: 

- "/new-for-you" route 
- stub app component
- stub test suite 
- associated feature flag

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[GRO-976]: https://artsyproduct.atlassian.net/browse/GRO-976?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Routes added to `routes.tsx` should have a corresponding cypress.js smoke test. See the `cypress/integration` folder for examples.
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 3a61c1853224175334a4855dd95d1436c4d20c78
</p>

File: src/v2/Apps/NewForYou/NewForYouApp.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React, { FC, useEffect } from "react"
+import { navigate } from "@reach/router"
```
Comment (damassi) [inline src/v2/Apps/NewForYou/NewForYouApp.tsx:2]: Weird, not sure how this got added to the app! We should use our `useRouter` hook, located in `v2/System/Router`. How did you stumble upon this?

File: src/v2/Apps/NewForYou/NewForYouApp.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import { useFeatureFlag } from "v2/System/useFeatureFlag"
+
+const NewForYouApp: FC = () => {
```
Comment (damassi) [inline src/v2/Apps/NewForYou/NewForYouApp.tsx:5]: We can do `export const NewForYouApp` an remove the bottom export

**Review by damassi — DISMISSED** [review]
Once the reach router import / usage is replaced with our app router this looks good to go 👍

File: src/v2/Apps/NewForYou/NewForYouApp.tsx (hunk 1 above)
Comment (laurabeth) [reply src/v2/Apps/NewForYou/NewForYouApp.tsx:2]: not sure. probably just a global search for the word router.

**Comment by The-Beez-Kneez** [comment]
We got some failing tests, once those are reserved I can merge it in :)

**Pull request merged**